src/StackExchange.Redis/Profiling/TimeSpanHelper.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Diagnostics;
+
+namespace StackExchange.Redis.Profiling
+{
+    /// <summary>
+    /// A helper class for dealing with Low/High Resolution Stopwatches and their ticks
+    /// </summary>
+    public class TimeSpanHelper
```

Comment:
should probably be `internal static class` ?

Reply:
Good shout. Have pushed an update :)